examples/sklearnex/patch_sklearn.py

```diff
@@ -31,7 +31,6 @@
 unpatch_sklearn()
 
 # Direct import of functions in way aligned with scikit-learn
```

Comment:
May need to take a closer look at this one - either remove comment as well or determine if there is a reason it is listed in example

Reply:
Sorry, this should not have been removed. Good catch!